### DISCUSSION TITLE
Rotation of Decorative objects In sub editor.

### DISCUSSION THREAD

**Discussion by Psyduckdeluxe** [FakeFishGames/Barotrauma#4920] [Ideas and suggestions]
The ability to rotate a lot more items in sub editor. Im not sure what more i can really add here! 

Personally i think this would be the biggest step for more vistually stunning subs. 
I know this has been mentined in a few other suggestions.  But i just think it deserves its own disgussion!

**Comment by mLuby** [comment]
Totally agree. This would unlock a lot more vanilla creations (item assemblies) to make subs feel unique and interesting.
